sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpEventBatch.cs

```diff
@@ -93,9 +92,11 @@ internal class AmqpEventBatch : TransportEventBatch
         ///
         /// <param name="messageConverter">The converter to use for translating <see cref="EventData" /> into the corresponding AMQP message.</param>
         /// <param name="options">The set of options to apply to the batch.</param>
+        /// <param name="measureUsingSequenceNumber">The sequence number to use when measuring event size.  If <c>null</c>, sequence numbers are not used in size calculations.</param>
```

Comment:
Could you explain a bit about the purpose of this argument? Some questions that arose while reviewing it:
- It appears to only be used as a measurement hint - if true, could this be eliminated and inferred based on whther a relevant feature is enabled that requires it?
- Perhaps I am missing something, but I don't see how it's used in the size calculation below
- Assuming it is needed and cannot be inferred, could it be a bool?

Reply:
The purpose is to allow the batch to measure size correctly, t

- It could be inferred, but I do not want to give the type responsibility for interpreting whether sequencing is needed; the client should make the decision and the batch just follow along.
- The end-to-end is not fully hooked up yet; this will be picked up by the `AmqpMessageConverter` and used to ensure that the size is calculated correctly when the correct sequence number is applied during publishing.
- It could, but I don't see much difference in approach, and I like having the batch be dumb.   Is there an advantage that you see to a flag over using what was provided?

Reply:
> * It could, but I don't see much difference in approach, and I like having the batch be dumb.   Is there an advantage that you see to a flag over using what was provided?

Only for clarity. `bool` contains all the needed information, and at first glance it's not clear what the actual value should represent.

Reply:
Its a fair point.  I'll take a look at swapping it over.

Reply:
Converted over.   You're right; I like that better.   Thanks for the kick.